components/mixins/experience-event/events/updateopportunity.schema.json

```diff
@@ -0,0 +1,65 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/events/updateopportunity",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Update Opportunity",
+  "type": "object",
+  "meta:conditionalField": "xdm:eventType",
+  "meta:conditionalValue": "opportunityEvent.updateOpportunity",
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "description": "Use for information regarding opportunity event update opportunity events.",
+  "definitions": {
+    "updateopportunity": {
+      "properties": {
+        "xdm:opportunityEvent": {
+          "title": "Opportunity Event",
+          "type": "object",
+          "properties": {
+            "xdm:opportunityID": {
+              "title": "Opportunity ID",
+              "type": "string",
+              "description": "Unique identifier for opportunity"
+            },
+            "xdm:dataValueChanges": {
+              "title": "Data Value Changes",
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "xdm:attributeName": {
+                    "title": "Attribute Name",
+                    "type": "string",
+                    "description": "Name of the attribute"
+                  },
+                  "xdm:newValue": {
+                    "title": "New Value",
+                    "type": "integer",
+                    "description": "New value of the attribute"
+                  },
+                  "xdm:oldValue": {
+                    "title": "Old Value",
+                    "type": "integer",
```

Comment:
This value can be in any data type. If we don't have such mechanism, we might choose only String data type here. Should have a discussion.